docs_src/design/adr/devops/0007-Release-Automation.md

```diff
@@ -12,14 +14,15 @@ EdgeX Foundry is a framework composed of microservices to ease development of Io
 
 ### Release Artifact Definition
 
-For the scope of Geneva release artifact types are defined as:
+For the scope of Hanoi release artifact types are defined as:
 
 - GitHub tags in the repositories.
 - Docker images in our Nexus repository and Docker hub.
-- Snaps in the Snapcraft store.
```

Comment:
Can we leave this line here and add an (*) asterisk with line 24 explaining being the explanation.